tests/sysmgr/mount.bats

```diff
@@ -25,19 +26,97 @@ load ../helpers/sysbox-health
   docker_stop "$syscont"
 }
 
-# verify sys container has a mount for /usr/src/linux-headers-<kernel>
-@test "kernel headers mounts" {
+# Verify that Ubuntu sys container has kernel-headers in the expected path.
+@test "kernel headers mounts (ubuntu)" {
 
   local kernel_rel=$(uname -r)
-  local syscont=$(docker_run --rm nestybox/alpine-docker-dbg:latest tail -f /dev/null)
+  local distro=$(get_distro)
 
-  docker exec "$syscont" sh -c "mount | grep \"/usr/src/linux-headers-${kernel_rel}\""
-  [ "$status" -eq 0 ]
+  local syscont=$(docker_run --rm ubuntu:bionic tail -f /dev/null)
 
-  if [ -n "$SHIFT_UIDS" ]; then
-    [[ "${lines[0]}" =~ "/usr/src/linux-headers-${kernel_rel} on /usr/src/linux-headers-${kernel_rel} type shiftfs".+"ro".+"relatime" ]]
-  else
-    [[ "${lines[0]}" =~ "on /usr/src/linux-headers-${kernel_rel}".+"ro".+"relatime" ]]
+  # Expected behavior will vary depending on the linux-distro running on the
+  # host (i.e. test-priv container).
+  if [[ "${distro}" == "centos" ]] ||
+      [[ "${distro}" == "fedora" ]] ||
+      [[ "${distro}" == "redhat" ]]; then
+
+      docker exec "$syscont" sh -c "mount | grep \"/usr/src/kernels/${kernel_rel}\""
+      [ "$status" -eq 0 ]
+
+      if [ -n "$SHIFT_UIDS" ]; then
+         [[ "${lines[0]}" =~ "/usr/src/kernels/${kernel_rel} on /usr/src/kernels/${kernel_rel} type shiftfs".+"ro".+"relatime" ]]
+      else
+        [[ "${lines[0]}" =~ "on /usr/src/kernels/${kernel_rel}".+"ro".+"relatime" ]]
+      fi
+
+      # Verify that /usr/src/linux-headers-$kernel_rel --> /usr/src/kernels/$kernel_rel
+      # softlink has been created in ubuntu's kernel-headers expected path.
+      docker exec "$syscont" sh -c "stat /usr/src/linux-headers-${kernel_rel} | egrep -q \"symbolic\""
+      [ "$status" -eq 0 ]
+
+ else
+      docker exec "$syscont" sh -c "mount | grep \"/usr/src/linux-headers-${kernel_rel}\""
+      [ "$status" -eq 0 ]
+
+      if [ -n "$SHIFT_UIDS" ]; then
+         [[ "${lines[0]}" =~ "/usr/src/linux-headers-${kernel_rel} on /usr/src/linux-headers-${kernel_rel} type shiftfs".+"ro".+"relatime" ]]
+      else
+        [[ "${lines[0]}" =~ "on /usr/src/linux-headers-${kernel_rel}".+"ro".+"relatime" ]]
+      fi
+
+      # Verify that no /usr/src/linux-headers-$kernel_rel --> /usr/src/kernels/$kernel_rel
+      # softlink has been created -- shouldn't be needed in this case as sysbox-runc/mgr
```

Comment:
This wording is a bit clearer I think:

"shouldn't be needed in this case..." -> "(even though it's not needed in this case ..."

Reply:
done.